src/device-registry/bin/www

```diff
@@ -150,17 +150,8 @@ const run = async() => {
                                 message: detail.message,
                                 key: detail.context.key,
                                 value: detail.context.value,
-                                path: detail.path,
                                 device_number: device_number ? device_number : undefined,
-                                device_id: device_id ? device_id : undefined,
                                 timestamp: timestamp ? timestamp : undefined,
-                                site_id: site_id ? site_id : undefined,
-                                battery: battery ? battery : undefined,
-                                network: network ? network : undefined,
```

Comment:
just to removed other details just to ensure that we do not clutter Slack notifs with uneeded details @BenjaminSsempala